pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -sv
```

Comment:
Did you want to enable -x as well? (This way the test fails immediately as opposed to running everything)

Reply:
Will add